MarsTestAutomation/Pages/LoginPage.cs

```diff
@@ -0,0 +1,35 @@
+using OpenQA.Selenium;
+using System;
+using MarsTestAutomation.Utilities;
+
+
+namespace MarsTestAutomation.Pages
+{
+    public class LoginPage
+    {
+        private string userName = "nandini.kusumpudi49@gmail.com";
+        private string password = "test@123";
+        public void LoginAction(IWebDriver driver)  
+        {
+            driver.Navigate().GoToUrl("http://localhost:5000/");
```

Comment:
i would move this to a constant helper that way you can change it from one place

Reply:
Done